src/js/model.js

```diff
@@ -9,42 +10,67 @@ class TodoObject {
         this.is_editable = false;
     }
 
+    todo() {
```

Comment:
@Chem-e what is the purpose of this method?

Reply:
it updates todo items in firebase 
as add()  only except object as parameter